.gitignore

```diff
@@ -3,6 +3,7 @@ log
 .DS_Store
 config/secrets.yaml
 config/validator.yaml
+config/ruby_gunnar.yaml
```

Comment:
this should not be in the repo. better to add config/private/ to the .gitignore file, and move your config there

Reply:
Fixed.